### ISSUE TITLE
inputAccessoryView space between keyboard

### ISSUE THREAD

**Issue by hapton** [SwiftUIX/SwiftUIX#428]
<img width="334" alt="error" src="https://github.com/SwiftUIX/SwiftUIX/assets/1235170/eae0d184-b35e-474d-83f2-ce9dbb0aa743">

This is a space between keyboard and toolbar in iOS 16.1
```
CocoaTextField(text: $text)
            .inputAccessoryView(
                HStack {
                    Spacer()
                    Button("Done") {

                    }
                   .padding(.trailing, 16)
                }
                .frame(height: 48)
            )
```
The same issue is in https://github.com/SwiftUIX/SwiftUIX/issues/351

**Comment by vmanot** [comment]
@hapton this should now be fixed. If my work with SwiftUIX helps you, please consider funding my OSS work on [Patreon](https://www.patreon.com/vmanot).

**Comment by hapton** [comment]
@vmanot  Fixed on the latest dev branch, please release the tag as soon as possible.Thanks.

**Comment by vmanot** [comment]
@hapton released 0.1.5.
